src/usb/usb_processing.c

```diff
@@ -58,35 +62,27 @@ static Packet _in_packet;
 // TODO: remove this global in future refactoring
 static struct queue* _global_queue;
```

Comment:
Could you remove this `_global_queue`? I guess it isn't used anymore.

Reply:
Fixed